database/mongodb/mongodb.def

```diff
@@ -0,0 +1,43 @@
+Bootstrap:library
+From: debian:9
+
+%environment
+	
+	#Setting up the hostname and port
+	HOSTNAME=127.0.0.1
+	PORT=27017
+	export HOSTNAME PORT
+	
+%help
+
+	Singularity container running MongoDB v4.0.3 Community Edition for Debian 9 (Stretch)
+
+%post
+	
+	#Setting up where data will reside
+	mkdir -p /data/db
+	#Give access to the folder
+	chmod -R 777 /data/db
```

Comment:
This is creating a `/data/db` in the container. This will be read-only in a SIF file.